pre-docs/writing_plans.md

```diff
@@ -523,3 +552,22 @@ not
 ```puppet
 without_default_logging { run_command('echo hi', $nodes) }
 ```
+
+## `puppetdb_query`
+
+TODO where should this go?
```

Comment:
It might make sense to start collecting a page documenting all the core Bolt functions. But for now I'd be tempted to put it after we introduce `get_targets` but before facts and vars on targets.

Reply:
I think that page will be strings generated pretty soon. 

get_targets and the facts/vars work seems more closely related since it's about how you interact with target state during a plan. The example I've given is target related but in general pdb quries are more general purpose.

Reply:
Ok. Then leaving them near the end seems fine.

Reply:
Do you want the TODO here still? Is that a note to Kate?

Reply:
That was my intention.